.storybook/preview.js

```diff
@@ -0,0 +1,23 @@
+// .storybook/preview.js
+
+import { MINIMAL_VIEWPORTS } from '@storybook/addon-viewport';
+
+const customViewports = {
+    tabletLandscape: {
+      name: 'Tablet Landscape',
+      styles: {
+        width: '1024px',
+        height: '963px',
+        },
+    },
+};
+
+export const parameters = {
+    layout: 'fullscreen',
+    viewport: {
+        viewports: {
```

Comment:
You may want to use our list of standard breakpoints [from Source](https://github.com/guardian/source/blob/main/src/core/foundations/src/breakpoints.ts).

Reply:
Good call  - added :)